pyproject.toml

```
@@ -50,8 +50,12 @@ path = "src/appsignal/__about__.py"
 
 [tool.hatch.build]
 sources = ["src"]
-exclude = ["src/scripts"]
-artifacts = ["src/appsignal/appsignal-agent"]
+exclude = ["src/scripts", "tests"]
```

```suggestion
exclude = ["src/scripts", "tests", "conftest.py"]
```
This test config file seems to be included as well.